src/stitches.config.ts

```diff
@@ -0,0 +1,301 @@
+      gray: '#EFF3F4',
+
+      lightBorder: '#DEDEDE',
+      lightGray: '#C3CDCF',
+      mediumGray: '#A7B0B4',
+      lightText: '#859296',
+      border: '#939EA1',
+      placeholder: '#99A2A5',
+      placeholderDark: '#7C898D',
+
+      primary: '#5E6F74',
+      text: '#516369',
+      secondary: '#56606f',
+
+      background: '#DFE7E8',
+      pacificBlue: '#00ABBF',
+      selected: '#31A5AC',
+      lightBlue: '#84C7CA',
+      darkBlue: '#5D9C9F',
+    },
+    space: {
+      px: '1px',
+      '2xs': '0.125rem', // 2px
+      xs: '0.25rem', // 4px
+      '1sm': '0.375rem', // 6px
+      sm: '0.5rem', // 8px
+      sm1: '0.625rem',
+      sm2: '0.75rem', // 8px
+      md: '1rem', // 16px
+      lg: '1.5rem', // 24px
+      xl: '2rem', // 32px
+      xl2: '2.25rem', // 32px
+      '1xl': '2.5rem', // 40px
+      '2xl': '3rem', // 48px
+      '3xl': '4rem', // 64px
+      '5xl': '5rem', // 80px
+
+      // non standard
+    },
+    sizes: {
+      max: 'max-content',
+      min: 'min-content',
+      full: '100%',
+      px: '1px',
+      '2xs': '0.125rem', // 2px
+      xs: '0.25rem', // 4px
+      '1sm': '0.375rem', // 6px
+      sm: '0.5rem', // 8px
+      md: '1rem', // 16px
+      lg: '1.5rem', // 24px
+      xl: '2rem', // 32px
+      '1xl': '2.5rem', // 40px
+      '2xl': '3rem', // 48px
+      '3xl': '4rem', // 64px
+      '4xl': '4.5rem',
+      '5xl': '5rem', // 80px
+    },
+    radii: {
+      1: '4px',
+      2: '6px',
+      3: '8px',
+      4: '12px',
+      round: '50%',
+      pill: '9999px',
+    },
+    fontSizes: {
+      1: '11px',
+      2: '12px',
+      3: '14px',
+      4: '16px',
+      5: '19px',
+      6: '21px',
+      7: '27px',
+      8: '35px',
+      9: '59px',
+
+      // non-standard
+      '5plus1px': '20px',
+      '5plus05px': '19.5px',
+    },
+    fontWeights: {
+      hairline: 100,
+      thin: 200,
+      light: 300,
+      normal: 400,
+      medium: 500,
+      semibold: 600,
+      bold: 700,
+      extrabold: 800,
+      black: 900,
+    },
+    lineHeights: {
+      normal: 'normal',
+      none: 1,
+      shorter: 1.25,
+      short: 1.375,
+      base: 1.5,
+      tall: 1.625,
+      tall2: 1.75,
+      taller: '2',
+      '3': '.75rem',
+      '4': '1rem',
+      '5': '1.25rem',
+      '6': '1.5rem',
+      '7': '1.75rem',
+      '8': '2rem',
+      '9': '2.25rem',
+      '10': '2.5rem',
+    },
+    letterSpacings: {
+      tighter: '-0.05em',
+      tight: '-0.025em',
+      normal: '0',
+      wide: '0.025em',
+      wider: '0.05em',
+      widest: '0.1em',
+    },
+    borderWidths: {},
+    borderStyles: {},
+    shadows: {},
+    zIndices: {},
+    transitions: {},
+  },
+  media: {
+    xs: MediaQueryKeys.xs,
+    sm: MediaQueryKeys.sm,
+    md: MediaQueryKeys.md,
+    lg: MediaQueryKeys.lg,
+    motion: '(prefers-reduced-motion)',
+    hover: '(any-hover: hover)',
+    dark: '(prefers-color-scheme: dark)',
+    light: '(prefers-color-scheme: light)',
+  },
+  utils: {
```

Comment:
let's seperate things that we expect to be shared between theme variations and things that would be changed, if for instance we allowed users to set primary color ect

Reply:
@exrhizo what do you mean by this? which part are you pointing here?